spec/features/categories/user_can_delete_category_from_index_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe "when user visits categories index page" do
+  scenario "they can click on delete and category is deleted" do
+
+    category1 = Category.create(title: "Technology")
```

Comment:
I'm glad you're doing all the active record stuff for practice, but I think in general for this test you are making too many categories, companies and jobs. 

You really only need one there to delete it. Now it would be cool to see a test that could be sure to delete the first and the last of the list of categories.